paddle/fluid/pybind/op_function.h

```diff
@@ -147,6 +147,61 @@ ConstructDuplicableOutput(const size_t num) {
   }
   return res;
 }
+
+static inline std::shared_ptr<imperative::VarBase> ConstructViewOutput(
+    const std::shared_ptr<imperative::VarBase>& input_var) {
+  PADDLE_ENFORCE_EQ(
+      input_var->Var().IsInitialized(), true,
+      platform::errors::InvalidArgument("Tensor %s has not been initialized!",
+                                        input_var->Name()));
+  PADDLE_ENFORCE_EQ(input_var->Var().IsType<framework::LoDTensor>() ||
+                        input_var->Var().IsType<framework::SelectedRows>(),
+                    true,
+                    platform::errors::InvalidArgument(
+                        "Type of Tensor[%s] must be LoDTensor or SelectedRows!",
+                        input_var->Name()));
```

Comment:
Please print the received type in the message, like 
"Type of Tensor[%s] must be LoDTensor or SelectedRows, but received XX"

Reply:
done